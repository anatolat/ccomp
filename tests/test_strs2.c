#include <stdio.h>
#include <string.h>


char strs[10][256];


int main() {
	for (int i = 0; i < 10; ++i) {
		strcpy(strs[i], "hello: ");
		for  (int j = 0; j < 10; ++j) {
			strs[i][j + 7] = '0' + j;
		}
	}

	for (int i = 0; i < 10; ++i) {
		printf("%s\n", strs[0]);
	}
}