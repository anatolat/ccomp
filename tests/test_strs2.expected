hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
hello: 0123456789
